README.md

```diff
@@ -1,9 +1,236 @@
 # CDK Framework
 
-The CDK Framework helps CDK authors build CDKs.
+The CDK Framework helps CDK authors build CDKs for the [Internet Computer](https://internetcomputer.org/).
 
 ## Core Concepts
 
 Start with the AST of your language and compile it into an Abstract Canister Tree (a concept from the CDK Framework). The CDK Framework will then create a Rust canister which can be compiled into Wasm and deployed to the IC.
```

Comment:
Flesh this out a bit to explain that the ACT will generate a `lib.rs` file, and explain that a Cargo.toml will need to be generated...or somehow explain that the lib.rs file can then be used to create a Rust canister